capstone/render.md

```diff
@@ -0,0 +1,244 @@
+# Deploying on Render
+
+Deployment is a very exciting step in the development process. Deployment is the process of employing a service to store your source code and data on their servers and providing a pathway in the form of the url. Render is platform used to host web applications. Render's services support many languages and it offers 3 month free deployments for small projects.
```

Comment:
Render is `a` platform used to host web applications.

Numbers smaller than 10 are typically written out: three months

Render's service supports many programming languages. Render offers three months of free deployment for small projects.